zerver/tests/test_event_queue.py

```diff
@@ -334,8 +334,8 @@ def test_wildcard_mention_in_muted_stream(self) -> None:
 
     def test_wildcard_mention_in_muted_topic(self) -> None:
         # Wildcard mentions in muted streams don't notify.
-        do_set_user_topic_visibility_policy(
-            self.user_profile,
+        bulk_do_set_user_topic_visibility_policy(
+            [self.user_profile],
```

Comment:
This feels a little ugly for these simpler callers; one option would be to offer a `do_set_user_topic_visibility_policy` wrapper that just calls the bulk version for you. I'll leave it up to you whether to do that; I'm not sure it really helps :)

Reply:
To me, it actually conveys a clear info that it does bulk_ operation with one user_profile.
@abhijeetbodas2001 your thoughts?

Reply:
It does convey the intention clearly, but having `do_set_user_topic_visibility_policy` as a very thin wrapper around the bulk function seems better to me just for the conciseness (especially since the "only one topic" case is not a one-off thing but used several times).

Reply:
Cool, updated the PR.

Check [commit-5](https://github.com/zulip/zulip/pull/24742/commits/48d35ef0d0d018c0a6b17c81d86e4e199802e62f#diff-6ce0345e26a4c98de5771cc6884431025ea9f65e8f1a6a1491341bf7d9603928R63) and [commit-6](https://github.com/zulip/zulip/pull/24742/commits/0230b8b321e932ab9b793c24cd525146a732cd5d#diff-26aa930b41caa2a7e31ee5622de9b7629fbe56f84c27a87315fa8189420d45e2R791) for the related changes.